bin/run_ess_sensors.py

```diff
@@ -21,12 +21,40 @@
 import asyncio
 import logging
 
-from lsst.ts.ess import sensors
+from lsst.ts.ess.sensors import SocketServer
 
 logging.basicConfig(
     format="%(asctime)s:%(levelname)s:%(name)s:%(message)s",
     level=logging.DEBUG,
 )
 
 
-asyncio.run(sensors.SocketServer.main())
+async def main():
+    """Main method that, when executed in stand alone mode, starts the socket
+    server.
+
+    The SocketServer automatically stops once the client exitst and at that
+    moment this script will exit as well.
+    """
+    logging.info("main method")
+    # An arbitrarily chosen port. Nothing special about it.
+    port = 5000
```

Comment:
You can allow a server to pick a free port, which is more robust -- you don't have to worry about any existing service using the fixed port.

(The socket server in ts_hexrotcomm supports that by having the user specify port=0, which I think is the standard convention. Itt then updates the port attribute once the port is known).

Reply:
Thanks, I am aware of that. However, it is a stand alone service that runs remotely so if I let the socket server choose a free port, the other side will not know which port it is. So I have to choose one beforehand. This should probably be done via some configuration YAML file but I haven't gotten around to implementing that yet.